### PR TITLE
Add Stripe to sponsor list

### PR DESCRIPTION
**Pull request by arpanlaha**
Adds Stripe to sponsor list. Additionally, sponsor logos now link to sponsor homepages and the size of sponsor images scales with screen width.

### CODE REVIEW

**Review by suewee — APPROVED** [review]
🎉

File: components/homepage/partnerSection.jsx [hunk 1]
```diff
@@ -12,27 +12,33 @@ export default function PartnerSection() {
             Our Sponsors
           </h1>
         </Row>
-        <Row>
-          <Col className="text-right">
+        <div className="sponsor-row">
+          <a href="https://www.imc.com/" alt="Stripe">
```
Comment (tffnd) [inline components/homepage/partnerSection.jsx:16]: `<a href="https://www.imc.com/" alt="IMC">`
change alt to IMC

**Review by tffnd — COMMENTED** [review]
`<a href="https://www.imc.com/" alt="IMC">`
line 16: change alt to IMC

Can you add the new Facebook and IMC logos too?
facebook_wordmark
imc_logo

**Comment by arpanlaha** [comment]
@tffnd I can do IMC, but the FB one has bad spacing and won't show up well on the website

**Pull request merged**